client/console/src/test/java/org/apache/syncope/client/console/AbstractTest.java

```diff
@@ -44,27 +53,75 @@
 import org.apache.syncope.common.lib.info.PlatformInfo;
 import org.apache.syncope.common.lib.info.SystemInfo;
 import org.apache.syncope.common.lib.to.DomainTO;
+import org.apache.syncope.common.lib.to.ExecTO;
+import org.apache.syncope.common.lib.to.JobTO;
 import org.apache.syncope.common.lib.to.UserTO;
+import org.apache.syncope.common.lib.types.JobType;
 import org.apache.syncope.common.rest.api.service.DomainService;
+import org.apache.syncope.common.rest.api.service.NotificationService;
+import org.apache.syncope.common.rest.api.service.ReportService;
 import org.apache.syncope.common.rest.api.service.SyncopeService;
+import org.apache.syncope.common.rest.api.service.TaskService;
+import org.apache.wicket.util.tester.FormTester;
 import org.apache.wicket.util.tester.WicketTester;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
 
 public abstract class AbstractTest {
 
     protected static Properties PROPS;
 
+    protected static SyncopeServiceClient service;
+    protected static TaskService taskService;
+    protected static ReportService reportService;
+    protected static NotificationService notificationService;
+
     public interface SyncopeServiceClient extends SyncopeService, Client {
     }
 
     @BeforeAll
     public static void loadProps() throws IOException {
         PROPS = new Properties();
-        try (InputStream is = AbstractTest.class.getResourceAsStream("/console.properties")) {
+        try ( InputStream is = AbstractTest.class.getResourceAsStream("/console.properties")) {
```

Comment:
Please remove this formatting-only change (space after parenthesis) and align with existing code, thanks.